Application/EdFi.Ods.WebService.Tests/Composites/Composites support hierarchical resources.feature

```diff
@@ -1,52 +0,0 @@
-﻿@Composites @HierarchicalComposite @Ignore
```

Comment:
Is there a any unused code in the composite steps that can be deleted?

Reply:
I looked. There are precious few steps used by the feature, and I could not find any steps that supported the language. So, I don't believe so.

Reply:
ok